RELEASE.md

```diff
@@ -2,6 +2,17 @@
 
 This document describes the Mimir release process as well as release shepherd responsibilities. Release shepherds are chosen on a voluntary basis.
 
+## Release schedule
+
+A new Grafana Mimir release is cut about every 6 weeks. The following table shows the past releases and tenantive date for upcoming ones.
+
+| Version | Date       | Release shepherd  |
+| ------- | ---------- | ----------------- |
+| 2.0.0   | 2022-03-20 | Marco Pracucci    |
+| 2.1.0   | 2022-05-16 | _To be announced_ |
+| 2.2.0   | 2022-06-27 | _To be announced_ |
+| 2.3.0   | 2022-08-08 | _To be announced_ |
```

Comment:
```suggestion
| 2.3.0 | 2022-08-08 | _To be announced_ |
| 2.2.0 | 2022-06-27 | _To be announced_ |
| 2.1.0 | 2022-05-16 | _To be announced_ |
| 2.0.0 | 2022-03-20 | Marco Pracucci |
```
Optional feedback to put the most current on top to eliminate scrolling over time.

Reply:
I would prefer to keep it in cronological order.